games-action/abuse/patches/abuse-0.8.patchset

```diff
@@ -0,0 +1,117 @@
+From 582dcee30889e4e5c3fac48e5e73be6ed74a027f Mon Sep 17 00:00:00 2001
+From: Yourself <user@shredder.home>
+Date: Tue, 25 Sep 2018 11:50:20 +0000
+Subject: removed reference to unused sys/shm.h
+
+
+diff --git a/src/net/tcpip.h b/src/net/tcpip.h
+index 000071c..a13ad9d 100644
+--- a/src/net/tcpip.h
++++ b/src/net/tcpip.h
+@@ -27,7 +27,6 @@
+ #   include <string.h>
+ #   include <sys/time.h>
+ #   include <sys/ipc.h>
+-#   include <sys/shm.h>
+ #   include <sys/socket.h>
+ #   include <unistd.h>
+ #   ifdef HAVE_BSTRING_H
+-- 
+2.19.0
+
+
+From 4f1008bbd75c664f4d3b0dcce20981e8f740925a Mon Sep 17 00:00:00 2001
+From: Yourself <user@shredder.home>
+Date: Tue, 25 Sep 2018 11:55:32 +0000
+Subject: replaced undeclared int32_t with int
```

Comment:
`int32_t` is defined in `<stdint.h>`, so please add an include for that instead.